openshift/templates/api/api-deploy.bc.prod.param

```diff
@@ -51,9 +50,9 @@ TAG_NAME=bc-prod
 DEMO_SITE=False
 # AGENT_SERVICE_NAME=agent
 # AGENT_ADMIN_PORT=8024
-MIN_REPLICAS=3
-# MAX_REPLICAS=16
-# CPU_REQUEST=10m
-# CPU_LIMIT=300m
-# MEMORY_REQUEST=10Mi
-# MEMORY_LIMIT=256Mi
+#MIN_REPLICAS=1
+#MAX_REPLICAS=1
+#CPU_REQUEST=1000m
+#CPU_LIMIT=2000m
+#MEMORY_REQUEST=512Mi
+#MEMORY_LIMIT=2Gi
```

Comment:
These values should be left to the previously set ones, as they have been tweaked for our production use-case.

Reply:
updated